Sources/SwiftSyntaxBuilder/gyb_generated/Tokens.swift

```diff
@@ -1,723 +1,693 @@
-//// Automatically Generated From Tokens.swift.gyb.
-//// Do Not Edit Directly!
-//===----------------------------------------------------------------------===//
-//
-// This source file is part of the Swift.org open source project
-//
-// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
-// Licensed under Apache License v2.0 with Runtime Library Exception
-//
-// See https://swift.org/LICENSE.txt for license information
-// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
-//
-//===----------------------------------------------------------------------===//
 
 import SwiftSyntax
 
 /// Namespace for commonly used tokens with default trivia.
-public extension TokenSyntax {
-  /// The `associatedtype` keyword
-  static var `associatedtype`: TokenSyntax {
+public extension TokenSyntax{
+  
+/// The `associatedtype` keyword
+static var `associatedtype`: TokenSyntax{
     SyntaxFactory.makeAssociatedtypeKeyword()
     .withTrailingTrivia(.spaces(1))
   }
-
-  /// The `class` keyword
-  static var `class`: TokenSyntax {
+  
+/// The `class` keyword
+static var `class`: TokenSyntax{
```

Comment:
This is an bug, where it don't respect the inject from the formatter.

Reply:
The reason is properly because the `withLeadingTrivia()` is overriding and not adding it.

Reply:
Fixing the indentation issue in a separate PR would be great. I think we can tackle this separately from this PR though.